README.md

```diff
@@ -219,6 +219,29 @@ Save the file as **.splunkrc** in the current user's home directory.
 
     Click **Yes**, then continue creating the file.
 
+### Create/Update a .conf file
+```javascript
+
+Async.chain([
+    function (done) {
+        // Fetch configurations
+        var configs = svc.configurations(namespace);
+        configs.fetch(done);
+    },
+    async function (configs, done) {
+        // Create a key-value map to store under a stanza
+        var keyValueMap = {}
+        keyValueMap[property1] = value1;
+
+        // If file/stanza doesn't exist, it will be created 
+        // else it will be updated.
+        configs.createAsync(filename, stanzaName, keyValueMap, done);
```

Comment:
Minor request - can we use actual example strings here so people are familiar with the format ("app.conf" vs "app", "[install]", vs "install")? A good example to use could be app.conf / [install] / is_configured as that's commonly used with setup pages, see https://docs.splunk.com/Documentation/Splunk/latest/admin/appconf#.5Binstall.5D

Reply:
Good catch.
Changes done.